README.md

```diff
@@ -206,7 +206,23 @@ containing directory instead of its contents, this code is not available.
 
 ## Notes on open data sources for Boston
 
+### https://recollect.net
+
+This is the new (third party) source for Boston's trash day address information.  
+Boston's trash day schedule here: https://www.boston.gov/trash-day-schedule
+labelled 'Powered by ReCollect'.
+
+trash_intent.py accesses the information at recollect.net
+
+jmartini noted on 02/13/2018 in issue #39: "The Boston digital team stated that the trash day info at data.boston.gov is often out of date. They are currently using a 3rd party called ReCollect to manage the newest trash day info."
+
 ### https://data.boston.gov
+
+**DEPRECATED** 
```

Comment:
I'm not sure we should call this whole site deprecated. It looks like there is still some valid data on there. It's unfortantely a case by case basis.

Reply:
Yes, let's get rid of that **DEPERECATED**